README.md

```diff
@@ -450,6 +450,11 @@ If you are instead using some custom python installation of Home Assistant then
   ```bash
   pip install zha-quirks==0.0.38
   ```
+After installing components with both methods above you need deleting the installed component by hand then  you is updating your HA installation or its override the new HA files and you can getting lib miss matching.  
```

Comment:
After installing components with the methods listed above you will need to delete the installed component by hand. To do this execute this command in the HA docker container CLI

Reply:
I  hope i have doing it right !!

Thanks for correcting my bad eng....